.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = tab
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
```

Comment:
I suggest keeping trailing white space for `.md` files.  
Use case: Returning to a new line is done by leaving a couple of white spaces at the end of the previous line.

Reply:
Fine! Working on it